### PR TITLE
Sign In page And Sign Up Page UI created

### PR DESCRIPTION
**Pull request by TamoorKhokhar**

### CODE REVIEW

File: src/pages/store.jsx [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    }
+    if (category === "") {
+      setError(true);
+    }
+    const newEntry = {
+      name: storeName,
+      categories: tags
+    };
+    const dataApi = await addStore(newEntry);
+    console.log(dataApi, "dataapi");
```
Comment (muneebk21) [inline src/pages/store.jsx:36]: console log

File: src/pages/store.jsx [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    dispatch(AddStore(dataApi));
```
Comment (muneebk21) [inline src/pages/store.jsx:37]: First check if API is returning correct response.

File: src/pages/store.jsx [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+  };
+  const [tags, setTags] = useState([]);
+  function handlekeyDown(e) {
+    if (e.key !== "Enter") return;
+    const value = e.target.value;
+    if (!value.trim()) return;
+    setTags([...tags, value]);
+    setcategory([]);
+  }
+  function RemoveTags(index) {
```
Comment (muneebk21) [inline src/pages/store.jsx:54]: Use camel case for function names.

File: src/pages/authentication/signIn.jsx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+import theme from "../../theme/theme";
+import Typography from "@mui/material/Typography";
+import { ThemeProvider } from "@emotion/react";
+import Header from "../../components/header";
+import Link from "@mui/material/Link";
+import SignInImage from "../../assets/images/SignIn.png";
+function SignIn() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  function SubmitForm(e) {
```
Comment (muneebk21) [inline src/pages/authentication/signIn.jsx]: Use camel case for function names.

File: src/pages/authentication/signUp.jsx [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+import Header from "../../components/header";
+import Link from "@mui/material/Link";
+import SignInImage from "../../assets/images/SignIn.png";
+
+function SignUp() {
+  const [userName, setUserName] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  function SubmitForm(e) {
```
Comment (muneebk21) [inline src/pages/authentication/signUp.jsx]: This is for the whole app, for methods/functions and variables use camel casing.

File: src/pages/authentication/signUp.jsx [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+                  />
+                  <Grid container>
+                    <Grid item xs>
+                      <Link href="#" variant="body2">
+                        Sign In As Admin
+                      </Link>
+                    </Grid>
+                    <Grid item>
+                      <Link href="signIn" variant="body2">
+                        {" Sign In"}
```
Comment (muneebk21) [inline src/pages/authentication/signUp.jsx]: Why do we need brackets and quotation marks here?

File: src/pages/authentication/signIn.jsx [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+                  />
+                  <Grid container>
+                    <Grid item xs>
+                      <Link href="#" variant="body2">
+                        Sign In As Admin
+                      </Link>
+                    </Grid>
+                    <Grid item>
+                      <Link href="/signUp" variant="body2">
+                        {"Don't have an account? Sign Up"}
```
Comment (muneebk21) [inline src/pages/authentication/signIn.jsx]: Why do we need brackets and quotation marks here? If they are necessary please explain, branch is not deployed for me to make an assumption.

**Review by MuneebKhan1996 — CHANGES_REQUESTED** [review]

File: src/pages/authentication/signIn.jsx [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+import { ThemeProvider } from "@emotion/react";
+import Header from "../../components/header";
+import Link from "@mui/material/Link";
+import SignInImage from "../../assets/images/SignIn.png";
+function SignIn() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  function SubmitForm(e) {
+    e.preventDefault();
+    console.log(email, password);
```
Comment (MuhAhs001) [inline src/pages/authentication/signIn.jsx]: remove consoles

File: src/pages/homePage.jsx [hunk 9]
```diff
@@ -9,7 +9,7 @@ import MorningTea from "../assets/images/MorningTea.png";
 import Lunch from "../assets/images/Lunch.png";
 import EveningTea from "../assets/images/EveningTea.png";
 import Modallunch from "../assets/images/ModalLunch.png";
-import ModalMorningTea from "../assets/images/ModalMorningTea.PNG";
+import ModalMorningTea from "../assets/images/ModalMorningTea.png";
 import ModalEveningTea from "../assets/images/ModalEveningtea.png";
```
Comment (MuhAhs001) [inline src/pages/homePage.jsx]: Why there is need to import these image and use that name in src you can directly that url in the src of image
or please use meaning full name 
it looks like name of components

Comment (TamoorKhokhar) [reply src/pages/homePage.jsx]: we have to import images in react when i give directly path in src image is not load on the screen .

**Review by MuneebKhan1996 — APPROVED** [review]

**Review by MuneebKhan1996 — APPROVED** [review]

**Pull request merged**